docs/UserGuide.md

```diff
@@ -14,11 +14,11 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `findvisor.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```

Comment:
```suggestion
1. Download the latest `findvisor.jar` from [here](https://github.com/AY2324S2-CS2103-F15-1/tp/releases).
```
Perhaps the URL of the JAR should be changed to our group's repo when our JAR eventually releases